charms/kfp-ui/src/components/pebble_components.py

```diff
@@ -50,16 +52,19 @@ def get_layer(self) -> Layer:
                             ).lower(),
                             "ARGO_ARCHIVE_ARTIFACTORY": "minio",
                             "ARGO_ARCHIVE_BUCKETNAME": "mlpipeline",
-                            "ARGO_ARCHIVE_LOGS": "false",
-                            "ARGO_ARCHIVE_PREFIX": "logs",
+                            "ARGO_ARCHIVE_LOGS": inputs.ARGO_ARCHIVE_LOGS,
+                            # Must have the same value as the `keyFormat` specified in the
+                            # `argo-workflow-controller-configmap` ConfigMap owned by
+                            # the `argo-controller` charm.
+                            "ARGO_KEYFORMAT": "artifacts/{{workflow.name}}/{{workflow.creationTimestamp.Y}}/{{workflow.creationTimestamp.m}}/{{workflow.creationTimestamp.d}}/{{pod.name}}",  # noqa E501
```

Comment:
Same as in https://github.com/canonical/argo-operators/pull/208#discussion_r1863307884

Although I understand that the CI is flaky, so am OK to keep this as it here

Reply:
I havent seen it break after #624 got merged so I committed and crossed my fingers :sweat_smile: